student.rb

```diff
@@ -0,0 +1,13 @@
+require_relative person
```

Comment:
- [ ] Kindly import the person class correctly.

  ```
  require_relative './person'
  ```

Reply:
Hi @julie-ify,

Thanks for the feedback. I have made the required changes.

Best Regards,
K. Inoulen Shanongba